zipline/data/bundles/quandl.py

```diff
@@ -391,6 +297,10 @@ def quantopian_quandl_bundle(environ,
                              cache,
                              show_progress,
                              output_dir):
+    log.warn(
+        'quantopian-quandl has been deprecated and '
```

Comment:
Should we use the `warnings` module for this?

Reply:
Also, if this is now deprecated, probably we should change the default `ingest` bundle to be the `quandl` bundle itself (in `zipline/__main__.py`.